src/server.ts

```diff
@@ -32,7 +32,7 @@ export const createApp = (
     removePinHeader,
     exposeHashedIdentity,
   }: AppConfig,
-  stampManager?: StampsManager,
+  stampManager?: BaseStampManager,
```

Comment:
No, please use the `StampsManager` interface instead of base class. If you want to understand why, please read for example this article: https://betterprogramming.pub/code-against-interfaces-not-implementations-37b30e7ab992

Reply:
`BaseStampManager` should not be used anywhere except where you `extends` it into the StampManager implementations.